pilot/lib/pilot/manager.rb

```diff
@@ -56,5 +56,13 @@ def fetch_app_identifier
       UI.verbose("App identifier (#{result})")
       return result
     end
+
+    def fetch_app_platform
```

Comment:
I'm still not seeing where this `fetch_app_platform` method is being used - line 10 uses `FastlaneCore::IpaFileAnalyser`'s implementation, but this doesn't seem to be called. Can we change the code at line 84 to call this method? Or else maybe delete this method?

Or am I missing something? 😄